README.md

```diff
@@ -1,134 +1,49 @@
-* **XMC-for-Arduino support for 'arm-linux-gnueabihf' only until version 1.1.**
-
-
-### Release packages from version 3.0.0 onwards
-
-Certain obsolete boards (see wiki) and non-functional libraries were removed from the board support package for the release version 2.0.0, alongwith some other major changes (see release notes). After version 3.0.0, the release index will not include library before version 2.0.0.
-
-![Board Manager](resources/wiki/image/Support_v2.png)
-### Known Issues
-
-* :warning: While using the pins connected to the **LEDs** configured as **INPUT**, there might be some abberation in behavior due to the presence of the series resistor of the LED, as it causes a voltage drop on the pin. In case of such an occurance, it is advised to desolder the series resistor and the LED and thereby using the pin as INPUT.  
-
-### Using PlatformIO IDE 
-
-- [What is PlatformIO?](http://docs.platformio.org/en/latest/what-is-platformio.html?utm_source=github&utm_medium=xmc-for-arduino)
-- [PlatformIO IDE](http://platformio.org/platformio-ide?utm_source=github&utm_medium=xmc-for-arduino)
-- [PlatformIO Core (CLI)](http://docs.platformio.org/en/latest/core.html?utm_source=github&utm_medium=xmc-for-arduino) (command line tool)
-- [Integration with Cloud and Desktop IDEs](http://docs.platformio.org/en/latest/ide.html?utm_source=github&utm_medium=xmc-for-arduino) -
-  Cloud9, Codeanywhere, Eclipse Che (Codenvy), Atom, CLion, Eclipse, Emacs, NetBeans, Qt Creator, Sublime Text, VIM, Visual Studio, and VSCode
-- [Project Examples](https://github.com/Infineon/platformio-infineonxmc/tree/master/examples)
-
-## Contributing and Third Party Libraries
-
-To contribute enhancements, fixes and the like see [CONTRIBUTINGS](./CONTRIBUTING.md).
-
-Third Party or external library maintainers see [Libraries](./Libraries.md). 
-    
-Also see [Wiki](https://github.com/Infineon/XMC-for-Arduino/wiki) for any additional information
-
-## Additional Contributors
+## Supported Microcontroller Boards
 
-* [Paul Carpenter](https://github.com/techpaul)
+<table style="text-align : center">
+    <tr>
+        <td><img src="docs/img/KIT_XMC14_2GO.png" width="200"></td>
+        <td><img src="docs/img/KIT_XMC11_BOOT_001.jpg" width="200"></td>
+        <td><img src="docs/img/KIT_XMC1400_ARDUINO.jpg" width="200"></td>
+        <td><img src="docs/img/KIT_XMC13_BOOT_001.jpg" width="200"></td>
+    </tr>
+    <tr>
+        <td><a href="https://www.infineon.com/cms/en/product/evaluation-boards/kit_xmc14_2go/?redirId=282145">KIT_XMC14_2GO</a></td>
+        <td><a href="https://www.infineon.com/cms/de/product/evaluation-boards/kit_xmc11_boot_001/">KIT_XMC11_BOOT_001</a></td>
+        <td><a href="https://www.infineon.com/cms/en/product/evaluation-boards/kit_xmc1400_arduino/">KIT_XMC1400_ARDUINO</a></td>
+        <td><a href="https://www.infineon.com/cms/de/product/evaluation-boards/kit_xmc13_boot_001/">KIT_XMC13_BOOT_001</a></td>
```

Comment:
Maybe... adding an extra row to avoid the scrolling? 
But then one board might be hanging there alone in the row. 
So maybe some boards pictures can be slightly smaller?

![image](https://github.com/user-attachments/assets/38e6dd41-4b9a-40de-872e-b2caba8f3524)

Reply:
I tried to make the pictures smaller, but it somehow still shows the scroll bar. Didn't like the approach with 3 rows